examples/simple-example/simple-example.pony

```diff
@@ -93,8 +93,11 @@ class Listener is TCPListenNotify
 
     try
       _out.print("client starting")
-      let ssl = _sslctx.client()?
-      TCPConnection(_auth, SSLConnection(ClientSide(_out), consume ssl), _host, _service)
+      TCPConnection(
+        _auth,
+        SSLConnection(ClientSide(_out), _sslctx.client()?),
+        _host,
+        _service)
```

Comment:
is this correct by the style guide? i thought it was supposed to only wrap extra things, not do 1 per line.